### PR TITLE
adding models

### PR DESCRIPTION
**Pull request by jonabr10**

### CODE REVIEW

File: server/models/user.model.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose')
+
+const User = new mongoose.Schema(
+    {
+        name: { type: String, required: true },
+        email: { type: String, reuqired: true, unique: true },
```
Comment (dabaquita) [inline server/models/user.model.js]: change `reuquired` to `required`

File: server/models/user.model.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        password: { type: String, required: true },
+        quote: { type: String }
+    },
```
Comment (dabaquita) [inline server/models/user.model.js]: add in a `id`

**Review by dabaquita — CHANGES_REQUESTED** [review]
Just fix up the line reviews please. 

Also, change the file to `user.js` instead :) Thanks!!

**Review by Resilliance — APPROVED** [review]
LGTM

**Review by Resilliance — APPROVED** [review]
LGTM

**Review by dabaquita — APPROVED** [review]

**Pull request merged**